hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -372,9 +362,7 @@ void notifyError(Object error) {
     }
 
     void notifyNormalResponse(Object value, int expectedBackups) {
-        if (value == null) {
-            value = NULL_RESPONSE;
-        }
+        value = value == null ? NULL_RESPONSE : value;
```

Comment:
I don't think this improves readability. In fact I think `value = value == null` is quite hard to read. If I want to write those single-line if-else I mostly use brackets to make give a hint, that there is a bool expr coming: `value = (value == null ? NULL_RESPONSE : value);`.